src/lightning_app/cli/lightning_cli.py

```diff
@@ -337,6 +339,75 @@ def stop() -> None:
 _main.add_command(create)
 
 
+@_main.command("ssh")
+@click.option(
+    "--app-id",
+    "app_id",
+    type=str,
+    default=None,
+    required=False,
+)
+@click.option(
+    "--component-name",
+    "component_name",
+    type=str,
+    default=None,
+    help="Specify which component to SSH into",
+)
+def ssh(app_id: str = None, component_name: str = None) -> None:
+    """SSH into a Lighting App."""
+    import inquirer
+
+    app_manager = _AppManager()
+    if app_id is None:
+        apps = app_manager.list_apps(phase_in=[V1LightningappInstanceState.RUNNING])
+        if len(apps) == 0:
+            raise click.ClickException("no running apps available.")
```

Comment:
What could we recommend to the user for the next step? Point them to the webpage with the list of the apps they could start? We could let them interactively choose an app to start using the inquirer, but one might have a very long list of stopped apps. We could list the top K recently used apps.

Reply:
For now I'll reword this to include a suggestion to start a Lightning App. Let's add more functionality once we see people using this feature.